bob-backend/src/pearl/group.rs

```diff
@@ -474,17 +474,40 @@ impl Group {
         String::from_utf8(hex).unwrap()
     }
 
+    pub(crate) async fn find_oldest_inactive_holder(&self) -> Option<Holder> {
+        let holders_lock = self.holders();
+        let holders = holders_lock.read().await;
+        let mut result: Option<Holder> = None;
+        for holder in holders.iter() {
```

Comment:
Is there a guarantee that `holders` are ordered by timestamp? If yes, then we can stop on the first holder. If no, then `close_unneeded_active_blobs` has a bug, thinking that they are ordered

Reply:
`close_unneded_active_blobs` uses custom sort on filtered holders, there is no assumption of holders ordering. Here we do not assume this too